catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/docs/kafka-component.adoc

```diff
@@ -580,3 +580,23 @@ from("kafka:my_topic?headerFilterStrategy=#myStrategy")
 `myStrategy` object should be subclass of `HeaderFilterStrategy` and must be placed in the Camel registry, either manually or by registration as a bean in Spring/Blueprint, as it is `CamelContext` aware.
 
 include::camel-spring-boot::page$kafka-starter.adoc[]
+
+== Consuming Avro messages from Kafka using the schema-registry
+*Since Camel 3.0.0*
+
+This route applies the schema and converts the Avro messages to POJOs for further processing:
+
+[source,java]
+----
+from(kafka("avro-events")
+    .brokers("localhost:9092")
+    .schemaRegistryURL("http://localhost:8081")
+    .specificAvroReader(true)
+    .valueDeserializer("io.confluent.kafka.serializers.KafkaAvroDeserializer")
+    .resolve(getContext()))
```

Comment:
You would not call resolve when using endpoint-dsl creating routes

Reply:
OK, let me remove resolve()

Reply:
Fixed in d5154d9